### ISSUE TITLE
mac address instead of IP address

### ISSUE THREAD

**Issue by astrojetson8** [YinHangCode/homebridge-mi-outlet#3]
Is it possible to use mac address instead of IP address? thanks

**Comment by coxjon** [comment]
Is this because of the changing IP addresses breaking Homekit?

I had a power outage and understandably the IP changed but came home today to a new IP too...

**Comment by astrojetson8** [comment]
yeah. I think this needs to be updated to used mac address or dynamically sourcing IP address.
